CLAUDE.md

```diff
@@ -241,7 +241,8 @@ Location: `.agents/steering/`
 ### QA Validation Is Required
 
 - Route to qa agent after feature implementation
-- Skip only for documentation-only changes (no code, config, or test changes)
+- Skip with `SKIPPED: docs-only` for documentation-only changes (no code, config, or test changes)
```

Comment:
@rjmurillo-bot there also needs to be adequate documentation for humans in CONTRIBUTING.md